packages/design-system/src/components/primitives/numeric-gesture-control.ts

```diff
@@ -165,9 +171,8 @@ export const numericScrubControl = (
         }, 150);
 
         targetNode.addEventListener("pointermove", handleEvent);
-        originalUserSelect =
-          targetNode.ownerDocument.documentElement.style.userSelect;
-        targetNode.ownerDocument.documentElement.style.userSelect = "none";
+        restoreUserSelect = setRootStyle(targetNode, "user-select", "none");
+        restoreContextMenu = preventContextMenu();
```

Comment:
Better to describe why we are preventing menu and how to reproduce. Same for select and other touches of global  things

Reply:
added comments